### PR TITLE
[Bug Bash] Clean up all unused components and files

### LINKED ISSUE #572: [Bug Bash][Frontend] Clean up all unused components and files

### PR DESCRIPTION
**Pull request by mxosman**
## Description of the change

Cleaning up all unused components and files in the Publisher app. Mostly styled components that either live in the common directory or are simply no longer used by any component.

## Related issues

Closes #572

### Development

**This box MUST be checked by the submitter prior to merging**:
- [x] **Double- and triple-checked that there is no Personally Identifiable Information (PII) being mistakenly added in this pull request**

These boxes should be checked by the submitter prior to merging:
- [ ] Tests have been written to cover the code changed/added as part of this pull request

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: publisher/src/components/DataViz/MetricsDataChart.tsx [hunk 1]
```diff
@@ -63,9 +61,11 @@ import {
   SystemName,
   SystemNameContainer,
   SystemsContainer,
-} from ".";
+} from "../MetricConfiguration";
+import { useSettingsSearchParams } from "../Settings";
+import ConnectedDatapointsView from "./ConnectedDatapointsView";
 
-export const MetricsView: React.FC = observer(() => {
+export const MetricsDataChart: React.FC = observer(() => {
```
Comment (mxosman) [inline publisher/src/components/DataViz/MetricsDataChart.tsx:68]: Changed this component name to be a bit more descriptive and moved it to the DataViz folder.

File: publisher/src/stores/StoreProvider.tsx [hunk 2]
```diff
@@ -17,7 +17,7 @@
 
 import React, { useContext, useEffect } from "react";
 
-import NoAuthConfigErrorPage from "../components/Error/NoAuthConfigErrorPage";
+import NoAuthConfigErrorPage from "../components/Auth/NoAuthConfigErrorPage";
```
Comment (mxosman) [inline publisher/src/stores/StoreProvider.tsx:20]: Moved this from a folder called `Errors` (it was the only component in there) to the `Auth` folder

File: publisher/src/components/DataViz/utils.ts [hunk 3]
```diff
@@ -281,59 +261,6 @@ export const transformData = (
...
-  data: Datapoint[],
-  isAnnual: boolean
-) => {
-  const mostRecentDate = data[data.length - 1]?.start_date;
-  if (mostRecentDate) {
-    const [, , month, year] = splitUtcString(mostRecentDate);
-    return `${!isAnnual ? `${month} ` : ""}${year}`;
-  }
-  return "N/A";
-};
```
Comment (mxosman) [inline publisher/src/components/DataViz/utils.ts:335]: All of these live in the common repo.

**Review by nichelle-hall — APPROVED** [review]
LGTM! Going for the code deletion award I see  👁

**Comment by mxosman** [comment]
> LGTM! Going for the code deletion award I see 👁

Thank you, Nichelle! Haha - I think Michelle won that one! Cleaning this up was honestly the reward - been aching to go through and purge!

**Pull request merged**